api/v1alpha1/storageconsumer_types.go

```diff
@@ -34,7 +34,8 @@ const (
 	// StorageConsumerStateFailed represents Failed state of StorageConsumer
 	StorageConsumerStateFailed StorageConsumerState = "Failed"
 	// StorageConsumerStateDisabled represents Disabled state of StorageConsumer
-	StorageConsumerStateDisabled StorageConsumerState = "Disabled"
+	StorageConsumerStateDisabled   StorageConsumerState = "Disabled"
+	StorageConsumerStateNotEnabled StorageConsumerState = "NotEnabled"
```

Comment:
This value needs a comment similar to the other ones

Reply:
done.